discovery-provider/src/challenges/challenges.dev.json

```diff
@@ -80,7 +80,8 @@
     "amount": 1,
     "active": true,
     "step_count": 2147483647,
-    "starting_block": 0
+    "starting_block": 0,
+    "weekly_pool": 20000
```

Comment:
just to be clear - this is implemented as a 20k cap _per_ challenge, and only for the audio matching challenges. not sure this is exactly what was discussed (thought it was 20k across _all_ challenges) but i think this totally gets us to where want to go.

Reply:
this is probably a placeholder number, but yeah i was thinking it would be per challenge, which gives us finer granularity, and if we wanted, we could always make another monthly_pool or something for each